src/lib/util/constants.ts

```diff
@@ -38,6 +38,12 @@ export const enum Emojis {
 	GreenTickSerialized = 's637706251253317669',
 	Loading = '<a:sloading:656988867403972629>',
 	RedCross = '<:redCross:637706251257511973>',
+	Calendar = '<:calendar_icon:1218607529702068294>',
+	SandsOfTime = '<:sandsoftime:1218608481565802587>',
```

Comment:
That's called an hourglass, not a sands of time. Is that what it's called in Spanish? Y'all nuts if so.

Sands of time is a [Prince of Persia game](https://en.wikipedia.org/wiki/Prince_of_Persia:_The_Sands_of_Time) with [some banger tunes](https://youtu.be/naPhZbhDxMQ)